src/widgets/hierarchical-menu/defaultTemplates.tsx

```diff
@@ -6,9 +6,13 @@ import { cx } from '../../lib/utils';
 import type { HierarchicalMenuComponentTemplates } from './hierarchical-menu';
 
 const defaultTemplates: HierarchicalMenuComponentTemplates = {
-  item({ url, label, count, cssClasses }) {
+  item({ url, label, count, cssClasses, isRefined }) {
+    const linkCssClasses = ([] as Array<string | undefined>)
+      .concat(cssClasses.link, isRefined ? cssClasses.selectedItemLink : [])
+      .filter(Boolean) as string[];
+
     return (
-      <a className={cx(cssClasses.link)} href={url}>
+      <a className={cx(linkCssClasses)} href={url}>
```

Comment:
```suggestion
      <a className={cx([cssClasses.link, isRefined && cssClasses.selectedItemLink])} href={url}>
```

Reply:
see https://github.com/algolia/instantsearch.js/blob/68f3df412468fc4536201deb6eaa8bca0554911b/src/lib/utils/cx.ts#L3

Reply:
`link` and `selectedItemLink` are typed as `string | string[] | undefined` so I need to concat them (or flatten the resulting array) to ensure we don't have arrays in arrays.

Reply:
could we call cx multiple times?

Reply:
Oh that's a good idea! 👏